ui/app/components/transit-key-action/rewrap.js

```diff
@@ -0,0 +1,14 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export default class RewrapComponent extends Component {
```

Comment:
One thing we may want to explore, to DRY (don't repeat yourself) this up is to make a component js file like `transit-key-action/base.js` with these actions defined, and then the rest of these component files can instead extend that base component. I know we've done similar patterns elsewhere, and I'm not totally sure it will work on glimmer components, but it's worth a try